jobs/rocoto/getic.sh

```diff
@@ -101,7 +101,14 @@ else # Pull chgres cube inputs for cold start IC generation
 fi
 
 # Move extracted data to ROTDIR
-if [ ! -d ${ROTDIR}/${CDUMP}.${yy}${mm}${dd}/${hh}/${COMPONENT} ]; then mkdir -p ${ROTDIR}/${CDUMP}.${yy}${mm}${dd}/${hh}/${COMPONENT} ; fi
+if [ ! -d ${ROTDIR}/${CDUMP}.${yy}${mm}${dd}/${hh}/${COMPONENT} ]; then
+  mkdir -p ${ROTDIR}/${CDUMP}.${yy}${mm}${dd}/${hh}/${COMPONENT}
+else
+  # Exception handling on case if ICS already exist
+  echo "The ICS target directory is not empty. Exiting job"
+  exit 0
+fi
```

Comment:
This is good if we only wanna check that the $COMPONENT folder exists and then exit. Thinking about this a bit more, I'm wondering if we should also check for the various IC files within before exiting. Thoughts @WalterKolczynski-NOAA ?